app/commands/feed/import_from_opml.rb

```diff
@@ -30,9 +30,20 @@ def find_or_create_group(group_name, user)
     def create_feed(parsed_feed, group, user)
       feed = user.feeds.where(**parsed_feed.slice(:name, :url))
                  .first_or_initialize
+      find_feed_name(feed, parsed_feed)
       feed.last_fetched = 1.day.ago if feed.new_record?
       feed.group_id = group.id if group
       feed.save
     end
+
+    def find_feed_name(feed, parsed_feed)
+      return unless parsed_feed[:name].nil? && feed.name.nil?
```

Comment:
It looks like these will be the same being that we're querying for `feed` based on the name from `parsed_feed`. Could we instead do:

```suggestion
      return if feed.name?
```

Reply:
It looks like it, but the `parsed_feed.slice(:name, :url)` is a bit misleading. If there's only one of the fields, it will only return that one:
```
3.4.1 :001 > parsed_feed = { name: "foo", url: "http://foo.com" }
 => {name: "foo", url: "http://foo.com"} 
3.4.1 :002 > parsed_feed.slice(:name, :url)
 => {name: "foo", url: "http://foo.com"} 
3.4.1 :003 > parsed_feed = { name: "foo" }
 => {name: "foo"} 
3.4.1 :004 > parsed_feed.slice(:name, :url)
 => {name: "foo"} 
3.4.1 :005 > parsed_feed = { url: "http://foo.com" }
 => {url: "http://foo.com"} 
3.4.1 :006 > parsed_feed.slice(:name, :url)
 => {url: "http://foo.com"} 
```
In other words, slice will not return nil fields, fields not found are just ignored.

I'll tweak the spec to make that clearer, I thought about it but forgot to do it